app/admin/user.rb

```diff
@@ -53,4 +53,14 @@
       row :updated_at
     end
   end
+
+  if ENV['IMPERSONATION_URL'].present?
```

Comment:
Have you discussed using Flipper here? Not a blocker.

Reply:
I like this idea too. Using a different url won't work anyway, since it won't match with the current controller and route